packages/odyssey/src/scss/components/_tab.scss

```diff
@@ -33,7 +33,6 @@
     &::before {
       content: '';
       position: absolute;
-      z-index: 1;
```

Comment:
I'd consider moving this change out of this PR into a separate one, as changes like this should be considered "breaking" and need to be documented independently of doc site updates.

Reply:
Will do before merge. Thanks Jordan!

Reply:
Added it back in.